scripts/fetch/dv_sites.R

```diff
@@ -0,0 +1,26 @@
+fetchTimestamp.dv_sites <- vizlab::alwaysCurrent
+
+#' @title Fetch appropriate daily value sites from NWIS
+#' 
+#' @param viz a vizlab object that depends on the date
+fetch.dv_sites <- function(viz){
+  library(dataRetrieval)
+  
+  deps <- readDepends(viz)
+  checkRequired(deps, "date")
+  date <- deps[["date"]][["date"]]
+  
+  hucs <- zeroPad(1:21, 2)
+  
+  sites <- c()
+  for(huc in hucs){
+    sites <- whatNWISdata(huc = huc, service = "dv", 
+                          startDate = date,
+                          parameterCd = "00060", 
+                          statCd = "00003") %>% 
+      .$site_no %>% 
```

Comment:
this is totally optional, but I think dplyr::pull is a cleaner way to write this. I haven't started updating my code to use it more, but we could here if we wanted to.